src/libraries/Microsoft.Extensions.Configuration.Binder/gen/Specs/TypeIndex.cs

```diff
@@ -109,14 +109,31 @@ public static string GetParseMethodName(ParsableFromStringSpec type)
         {
             Debug.Assert(type.StringParsableTypeKind is not StringParsableTypeKind.AssignFromSectionValue);
 
-            string displayString = type.DisplayString;
+            if (type.StringParsableTypeKind is StringParsableTypeKind.ByteArray)
```

Comment:
I had to refactor this method somewhat to make it account for using fully qualified names.